vcr/issuer/issuer.go

```diff
@@ -119,19 +125,23 @@ func (i issuer) Issue(ctx context.Context, credentialOptions vc.VerifiableCreden
 	if publish {
 		// Try to issue over OIDC4VCI if it's enabled and if the credential is not public
 		// (public credentials are always published on the network).
-		if i.oidcIssuer != nil && !public {
-			err := i.issueUsingOIDC4VCI(ctx, *createdVC)
-			if err == nil {
+		if i.openidIssuerFn != nil && !public {
+			success, err := i.issueUsingOpenID4VCI(ctx, *createdVC)
+			if err != nil {
+				// An error occurred, but it's not because the wallet/issuer doesn't support OpenID4VCI.
+				log.Logger().
+					WithField(core.LogFieldCredentialID, createdVC.ID.String()).
+					WithError(err).
+					Warnf("Could publish credential over OIDC4VCI, fallback to publish over Nuts network")
+			} else if success {
 				log.Logger().
 					WithField(core.LogFieldCredentialID, createdVC.ID.String()).
 					Info("Published credential over OIDC4VCI")
 				return createdVC, nil
-			} else if !errors.Is(err, vdr.ErrServiceNotFound) {
-				// An error occurred, but it's not because the DID doesn't support OIDC4VCI. Fallback to publishing over Nuts network.
+			} else {
 				log.Logger().
 					WithField(core.LogFieldCredentialID, createdVC.ID.String()).
-					WithError(err).
-					Warnf("Could publish credential over OIDC4VCI, fallback to publish over Nuts network")
+					Info("Wallet or issuer does not support OpenID4VCI, fallback to publish over Nuts network")
```

Comment:
This should only happen if the wallet doesn't support it?

Reply:
Or when the issuer doesn't support it, meaning it didn't configure the base URL in its DID document, and it can't be resolved using the TLS certificate.